packages/picasso/src/Tabs/Tabs.tsx

```diff
@@ -21,6 +21,9 @@ export interface Props
 
   /** The tabs orientation (layout flow direction). */
   orientation?: 'horizontal' | 'vertical'
+
+  /** Determines additional display behavior of the tabs */
+  variant?: TabsProps['variant']
```

Comment:
I checked the [MUI docs](https://v4.mui.com/api/tabs/#props) and there are three options available `'fullWidth'
| 'scrollable' | 'standard'`. I think right now we want to support only `fullWidth | scrollable`. It might be confusing for users.

```suggestion
  variant?:  'fullWidth' | 'scrollable'
```